src/source_drivers/generic/services/generic_point_listener.py

```diff
@@ -0,0 +1,97 @@
+import logging
+from json import loads as json_loads, JSONDecodeError
+from sqlalchemy.orm.exc import ObjectDeletedError
+
+from src import db
+from src.source_drivers.generic.services import GENERIC_SERVICE_NAME
+from src.event_dispatcher import EventDispatcher
+from src.services.event_service_base import EventServiceBase
+from src.services.mqtt_client.mqtt_client_base import MqttClientBase
+from src.ini_config import *
+from src.models.point.model_point import PointModel
+from src.models.point.model_point_store import PointStoreModel
+
+logger = logging.getLogger(__name__)
+
+MQTT_CLIENT_NAME = 'rubix_points_generic_point'
+
+GENERIC_POINT_MQTT_TOPIC = 'rubix/points/generic/cov'
+GENERIC_POINT_MQTT_TOPIC_EXAMPLE = f'{GENERIC_POINT_MQTT_TOPIC}/point_name/device_name/network_name'
```

Comment:
`GENERIC_POINT_MQTT_TOPIC_EXAMPLE = f'{GENERIC_POINT_MQTT_TOPIC}/<point_name>/<device_name>/<network_name>'`

This would be more meaningful.